Source/Editor/Windows/Assets/JsonAssetWindow.cs

```diff
@@ -165,6 +239,62 @@ protected override void OnAssetLoaded()
             base.OnAssetLoaded();
         }
 
+        private void OpenOptionsContextMenu()
+        {
+            if (_optionsCM != null && _optionsCM.ContainsFocus)
+                return;
+            
+            _optionsCM = new ContextMenu();
+            _optionsCM.AddButton("Copy type name", () => Clipboard.Text = Asset.DataTypeName);
+            _optionsCM.AddButton("Copy Asset data", () => Clipboard.Text = Asset.Data);
```

Comment:
Let's change `Asset` to `asset` lowercase to maintain the case in the context menu (I'm aware it could be already made that way in other Editor parts)